### ISSUE TITLE
Global venue database

### ISSUE THREAD

**Issue by jace** [hasgeek/funnel#448]
Venues are currently defined within each project and replicated when a new event is held at the same venue. This is wasteful.

We previously justified this on the pretext that (a) project owners must have complete control, and (b) venue details change over time, so they need to be kept updated without changing the facts for older events. A previous attempt at a global venue database in the Hacknight project threw up a third problem: ownership of the venue record. We did not have a wiki-like venue data update mechanism.

We have the necessary tools now:

1. The membership models (in #401) show us how to create replaceable models. (They don't have a solution for child models, which we will have to solve here. If a venue is refreshed, what happens to the rooms that were linked to the previous version of the venue? Do they have to be refreshed as well?)

2. We have clearly defined profiles that can own venues. This works for corporate venues that are direct users of the platform. We'll need "ownerless" venues in addition that are maintained by platform moderators.

3. Platform moderators are now a clearly defined membership role, obviating the need for a crude siteadmin mechanism.

TL;DR: Move the Venue model from Project to Profile scope, allow any project to use any venue in the database, and remove the dupes. (In a future ticket we can add access control for projects linking to venues.)

**Comment by jace** [comment]
Venues are no longer a priority in the pandemic era. This problem can be revisited with fresh eyes when offline events pick up again. Closing for now.
